### PR TITLE
GH-2189 Adds optional 'info' button to MegaChart

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-2189

@GordonSmith @jprichard Please review

Signed-off-by: jbrundage <jbrundage372@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage It doesn't appear to do anything?

**Comment by jbrundage** [comment]
It doesn't. It's an anchor for binding click events on the app layer. This change simply allows showing/hiding this feature via DDL.

File: src/composite/MegaChart.js [hunk 1]
```diff
@@ -51,6 +51,8 @@
     MegaChart.prototype.publish("showCSV", true, "boolean", "Show/Hide CSV button", null, { tags: ["Basic"] });
     MegaChart.prototype.publish("showMaximize", true, "boolean", "Show/Hide Maximize button", null, { tags: ["Basic"] });
     MegaChart.prototype.publish("toolbarShowLegend", false, "boolean", "Show/Hide Legend button", null, { tags: ["Basic"] });
+    MegaChart.prototype.publish("showInfoButton", true, "boolean", "Show/Hide Info button in toolbar", null, { tags: ["Basic"] });
```
Comment (GordonSmith) [inline src/composite/MegaChart.js]: Should default to false.

File: src/composite/MegaChart.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+.composite_MegaChart-Info,
+.composite_MegaChart-Info *,
```
Comment (GordonSmith) [inline src/composite/MegaChart.css]: Should these changes be here?

Comment (jbrundage) [inline src/composite/MegaChart.css]: Yes, I'm using css to default the font family to "FontAwesome." The same way the maximize button font family defaults.

**Pull request merged**